app/assets/src/components/views/phylo_tree/PhyloTreeByPathCreation.jsx

```diff
@@ -6,6 +6,8 @@ import axios from "axios";
 import DataTable from "../../visualizations/table/DataTable";
 import Moment from "react-moment";
 import PhyloTreeChecks from "./PhyloTreeChecks";
+import SearchBox from "../../ui/controls/SearchBox";
+import LoadingSpinner from "../../ui/icons/LoadingSpinner";
 
 const MinNumberOfSamples = 4;
 class PhyloTreeByPathCreation extends React.Component {
```

Comment:
You can now change this awkward name to `PhyloTreeCreation`!

Reply:
ok done